server/src/lib/routes/verify/get_session_cookie.ts

```diff
@@ -58,10 +57,9 @@ export default function (req: Express.Request, res: Express.Response,
     groups = authSession.groups;
 
     if (!authSession.userid) {
-      reject(new Exceptions.AccessDeniedError(
+      return reject(new Exceptions.AccessDeniedError(
```

Comment:
Initially you invoked the reject and then returned afterwards:
https://github.com/clems4ever/authelia/blob/e7ad831d4d162637bc0419d120db02defbdc02f1/server/src/lib/routes/verify/get_session_cookie.ts#L61-L64

We're simply moving the return to return the reject, to keep it inline with what you've done further below:
https://github.com/clems4ever/authelia/blob/e7ad831d4d162637bc0419d120db02defbdc02f1/server/src/lib/routes/verify/get_session_cookie.ts#L78-L80

You need to return there otherwise you'll carry on when the required userid is missing.